shared_model/builders/protobuf/builder_templates/transaction_template.hpp

```diff
@@ -125,14 +126,15 @@ namespace shared_model {
         });
       }
 
-      auto createdTime(interface::types::TimestampType created_time) const {
+      auto createdTime(
+          const interface::types::TimestampType &created_time) const {
         return transform<CreatedTime>([&](auto &tx) {
           tx.mutable_payload()->mutable_reduced_payload()->set_created_time(
               created_time);
         });
       }
 
-      auto quorum(interface::types::QuorumType quorum) const {
+      auto quorum(const interface::types::QuorumType &quorum) const {
```

Comment:
```QuorumType``` is an ```unsigned int```, and it should no be passed by reference, if you are not going to change it. The same thing happens also in 278 line